src/config/Util.hxx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2003-2017 The Music Player Daemon Project
+ * http://www.musicpd.org
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef MPD_CONFIG_UTIL_HXX
+#define MPD_CONFIG_UTIL_HXX
+
+class ServerSocket;
+
+void
+ServerSocketAddGeneric(const char *address, unsigned int port, ServerSocket &server_socket);
```

Comment:
The `ServerSocket` should be the first parameter here - by convention again, and because this is some kind of `this` pointer which is also the first parameter passed to a method.